src/leapr.hpp

```diff
@@ -8,50 +8,70 @@
 #include <range/v3/all.hpp>
 #include <variant>
 #include "endout/endout.h"
+#include <string>
 using namespace njoy;
 #include "lipservice.hpp"
 
 
-auto finalLEAPR( nlohmann::json jsonInput ){
-  // Do we have a secondary scatterer?
+
+namespace njoy {
+namespace LEAPR {
+
+class LEAPR{
+
+public:
+void operator()( const nlohmann::json& jsonInput,
+                 std::ostream& output,
+                 std::ostream& error,
+                 const nlohmann::json& ){
+
+  //output << "Input arguments:\n" << jsonInput.dump(2) << std::endl;
```

Comment:
It's not required, but it *very* helpful to the end user to know that NJOY is reading in the same values they intended to include. I would prefer that this comment be included (i.e., not a comment).

Reply:
Oh gosh, you're right. I commented that out so that it wouldn't print to the screen while I was testing things, but forgot to add it back in. Thank you for pointing that out, I'll change that now